src/cluster.c

```diff
@@ -233,6 +249,12 @@ static void OnResponseArrived(struct redisAsyncContext* c, void* a, void* b){
         return;
     }
     Node* n = (Node*)b;
+    if(reply->type == REDIS_REPLY_ERROR && strncmp(reply->str, CLUSTER_ERROR, strlen(CLUSTER_ERROR)) == 0){
```

Comment:
`strncmp(reply->str, CLUSTER_ERROR, strlen(CLUSTER_ERROR))` vs `strcmp(reply->str, CLUSTER_ERROR)`

Reply:
@rafie only the prefix need to be compared